### PR TITLE
TDD: Check that attempt exists in the English dictionary

### PR DESCRIPTION
**Pull request by elly-eht**

### CODE REVIEW

**Comment by adote2020** [comment]
Ok for the merge

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+requests = "*"
```
Comment (AurelienDimicoli) [inline Pipfile:7]: Tu devrais renseigner la version !

**Review by AurelienDimicoli — COMMENTED** [review]
1 comment :)

File: Pipfile (hunk 1 above)
Comment (elly-eht) [reply Pipfile:7]: indeed :)

**Pull request merged**